FWCore/Integration/test/ESTestAnalyzers.cc

```diff
@@ -213,10 +214,28 @@ namespace edmtest {
                                         << ": Data values = " << dataJ->value();
   }
 
+  class ESTestAnalyzerL : public edm::stream::EDAnalyzer<> {
+  public:
+    explicit ESTestAnalyzerL(edm::ParameterSet const& iConfig)
+        : edToken_(consumes(iConfig.getParameter<edm::InputTag>("src"))), esToken_(esConsumes()) {}
+    void analyze(const edm::Event&, const edm::EventSetup&) override;
+
+  private:
+    edm::EDGetTokenT<IntProduct> edToken_;
```

Comment:
Why the `edToken_` since it is never actually used in `analyze`?

Reply:
Seems strange not to actually get the data as this is testing for case where the data is available but was not retrieved.

Reply:
Consumption with `@currentProcess` was sufficient to trigger the bug. I can certainly add a use of the `IntProduct` as well.

Reply:
Yes, please add a check on `IntProduct` as well.